app/controllers/MovementsController.scala

```diff
@@ -64,7 +74,6 @@ class MovementsController @Inject()(
                 InternalServerError
```

Comment:
We'll need some logging to understand what the error was because it may have been on the insert or post